coderd/coderd.go

```diff
@@ -611,6 +626,14 @@ func New(options *Options) *API {
 					r.Use(httpmw.ExtractOAuth2(options.OIDCConfig, options.HTTPClient, oidcAuthURLParams))
 					r.Get("/", api.userOIDC)
 				})
+				// This is an authenticated route. It handles converting a user
+				// from Password based authentication to Oauth
+				r.Route("/convert-login", func(r chi.Router) {
```

Comment:
Could this be on the `/users/<id>` route instead? Seems possible we'd want to allow users to convert other users login type in the future.